librz/debug/dmap.c

```diff
@@ -82,16 +78,20 @@ static void print_debug_map_line(RzDebug *dbg, RzDebugMap *map, ut64 addr, const
 
 RZ_API void rz_debug_map_list(RzDebug *dbg, ut64 addr, const char *input) {
 	int i;
-	bool notfirst = false;
 	RzListIter *iter;
 	RzDebugMap *map;
+	PJ *pj;
 	if (!dbg) {
 		return;
 	}
 
 	switch (input[0]) {
 	case 'j': // "dmj" add JSON opening array brace
-		dbg->cb_printf ("[");
+		pj = pj_new ();
```

Comment:
this should be moved out so pj is initialized and available in each switch.
`if (input[0] == 'j') { init pj }`

Reply:
@wargio it's used only once in this function. I don't think it's required.